### PR TITLE
feat(FEC-9732): DFP DAI - Studio Configuration

### PR DESCRIPTION
**Pull request by RoyBregman**
Added IMA-DAI to studio configuration

Also updated the IMA google SDK link as it was not valid

Solves FEC-9732

### CODE REVIEW

File: app/js/services/v3Properties.json [hunk 1]
```diff
@@ -395,6 +395,19 @@
...
+      },
+      "imadai": {
+        "label": "IMA-DAI",
+        "type": "featuremenu",
+        "description": "The Google IMA DAI SDK for HTML5 allows developers to make a stream request for ad and content video and get the combined (stitched) video stream in an HTML5 video environment. ",
+        "model": "config.plugins.imadai",
+        "componentName": "playkit-ima-dai",
+        "children": [
+          {
+            "label": "Dummy",
```
Comment (yairans) [inline app/js/services/v3Properties.json:407]: try to fix the empty children issue

File: app/js/services/v3Properties.json [hunk 2]
```diff
@@ -395,6 +395,19 @@
             "initvalue": "SELECT_EXACT_MATCH"
           }
         ]
+      },
+      "imadai": {
+        "label": "IMA-DAI",
+        "type": "featuremenu",
+        "description": "The Google IMA DAI SDK for HTML5 allows developers to make a stream request for ad and content video and get the combined (stitched) video stream in an HTML5 video environment. ",
+        "model": "config.plugins.imadai",
```
Comment (yairans) [inline app/js/services/v3Properties.json:403]: you don't need to update the model as you just want to bundle the plugin with no config
see the receiver as an example of this

File: app/js/services/v3Properties.json (hunk 1 above)
Comment (RoyBregman) [reply app/js/services/v3Properties.json]: fixed

File: app/js/services/v3Properties.json (hunk 2 above)
Comment (RoyBregman) [reply app/js/services/v3Properties.json]: fixed

**Pull request merged**